DataProcessing/Spark.nuspec

```diff
@@ -31,9 +31,10 @@
 <file src="**\kafka-clients-2.0.0.jar" target="lib" />
 <file src="**\spark-streaming-kafka-0-10_2.11-2.4.0.jar" target="lib" />
 <file src="**\azure-sqldb-spark-1.0.2.jar" target="lib" />
-<file src="**\hadoop-azure-2.7.0.jar" target="lib" />
```

Comment:
we need to move this to 3.2.  probably want to do that in the same checkin

Reply:
CG suggested 2.7.3 version of hadoop-azure.. After upgrading the critical alert went away on run CG on the PR branch